app/views/api/projects/dependencies.json.jb

```diff
@@ -0,0 +1,30 @@
+dependencies_fields = {
+  dependencies_for_version: @version.number,
+  dependencies: (@version.dependencies.includes(:project) || []).
+    map {|dependency| DependencySerializer.new(dependency) }.
+    as_json
+}
+
+case @subset
+when "default"
+  @project.
+    slice(Project::API_FIELDS).
+    merge({
+      versions: @project.versions.map do |v|
```

Comment:
If we wanna squeeze a little extra out of this we could pluck all the necessary fields. We'd have to twiddle repository sources after the fact but could be worth it. Something along the lines of https://github.com/librariesio/libraries.io/blob/100cafcc0bae517dd43b15ea51a7ba00fca7bd7f/app/serializers/optimized_project_serializer.rb#L83